backend/services/transactions.js

```diff
@@ -0,0 +1,34 @@
+import Transaction from '../models/transactions.js';
+
+export default class TransactionsService {
+  static getTransactions(accountId) {
+    const transactions = Transaction.find({ accountId });
+    return transactions;
+  }
+
+  static getTransaction(transactionId) {
+    const transaction = Transaction.findOne({ _id: transactionId });
```

Comment:
You return promise. It's better to add async befor method name (for all this methods)